Symfony/CS/Tests/Config/ConfigTest.php

```diff
@@ -79,4 +79,42 @@ public function testThatMutatorHasFluentInterface()
 
         $this->assertSame($config, $config->setCacheFile('some-directory/some.file'));
     }
+
+    public function testAddRules()
+    {
+        $config = new Config();
+        $config->setRules(array(
+            'foo' => true,
+            'bar' => true,
+        ));
+
+        $config->addRules(array(
+            'foo' => false,
+            'something' => array('with' => 'array'),
+        ));
+
+        $this->assertSame(array(
+            'foo' => false,
+            'bar' => true,
+            'something' => array('with' => 'array'),
+        ), $config->getRules());
+    }
+
+    public function testAddRule()
+    {
+        $config = new Config();
+        $config->setRules(array(
+            'foo' => true,
+            'bar' => true,
+        ));
+
+        $config->addRule('foo', false);
```

Comment:
I can throw an exception instead if you want.

If not, I let your the liberty to close this PR and the related issue.